bencher/plotting/plot_collection.py

```diff
@@ -86,14 +87,24 @@ def gather_plots(
         Args:
             bench_cfg (BenchCfg): A config of the input vars
             rv (ParametrizedSweep): a config of the result variable
-            plt_cnt_cfg (PltCntCfg): A config of how many input types there are"""
+            plt_cnt_cfg (PltCntCfg): A config of how many input types there are
+        Raises:
+            ValueError: If the plot does not inherit from a pn.viewable.Viewable type
+        """
 
         tabs = pn.Accordion()
         for plt_fn in self.plotters.values():
             plots = plt_fn(PlotInput(bench_cfg, rv, plt_cnt_cfg))
-            for plt_instance in plots:
-                logging.info(f"plotting: {plt_instance.name}")
-                tabs.append(plt_instance)
+            if plots is not None:
+                if type(plots) != list:
```

Comment:
What's the motivation for this change? Is this trying to catch the case where plots has a length of 1?

Reply:
before all plots would be contained in a list. Now the plots are either a single plot or potentially a list of plots.  This just sets the plots to the most compatible format (a list of plots) for the rest of the processing.